Web/Edubase.Web.UI/Helpers/HtmlHelperExtensions.cs

```diff
@@ -73,7 +74,8 @@ public static MvcHtmlString ValidationMessageNested(this HtmlHelper htmlHelper,
                     {
                         foreach (var error in htmlHelper.ViewData.ModelState[modelName].Errors)
                         {
-                            htmlHelper.ViewData.ModelState.AddModelError(fullFieldName, error.ErrorMessage);
+                            var itemSplit = Regex.Replace(error.ErrorMessage, "([a-z])([A-Z])", "$1 $2");
```

Comment:
Could this line be put into a method that has a name that describes what is happening. It would make the code more readable.